plugins/modem-manager/modem-manager.quirk

```diff
@@ -78,3 +78,16 @@ ModemManagerFirehoseProgFile = prog_firehose_sdx6x.elf
 [PCI\VID_1EAC&PID_1007]
 Summary = Quectel RM520 firehose prog file
 ModemManagerFirehoseProgFile = prog_firehose_sdx6x.elf
+
+# Fibocom FM101
+[USB\VID_2CB7&PID_01A2]
+Summary = Fibocom Fibocom FM101-GL Module
```

Comment:
Fibocom Fibocom?

Reply:
This is the I got using the `lsusb` query

Here are some of the results I get using `lsusb-v` :

```shell
ght@ght-Latitude-7640:~$ lsusb -v.

Bus 004 Device 030: ID 2cb7:01a2 Fibocom Fibocom FM101-GL Module
Couldn't open device, some information will be missing
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               3.20
  bDeviceClass            0
  bDeviceSubClass         0
  bDeviceProtocol         0
  bMaxPacketSize0         9
  idVendor           0x2cb7 Fibocom
  idProduct          0x01a2
  bcdDevice            5.04
  iManufacturer           1 Fibocom Wireless Inc.
  iProduct                2 Fibocom FM101-GL Module
  iSerial                 3 b6c08415
  bNumConfigurations      1
  Configuration Descriptor:
```

It looks like `idVendor` and `iProduct` are concatenated.

I'll modify it:

```
Summary = Fibocom FM101-GL Module
```

Thanks.